test/runnable/test22016.d

```diff
@@ -0,0 +1,20 @@
+// https://issues.dlang.org/show_bug.cgi?id=22016
+
+/*
+TEST_OUTPUT:
+---
+---
+*/
```

Comment:
Empty output sections are the default nowadays

Reply:
I know, but I remembered @ibuclaw telling me at some point that it's good to be explicit.

Reply:
That was probably before I/someone else made the change to the test runner to complain about _any_ untracked output.